Makefile

```diff
@@ -86,6 +86,23 @@ testbuild:
 	  $(GO) test $(GOFLAGS) -tags '$(TAGS)' -o "$$DIR"/"$$OUT" -ldflags '$(LDFLAGS)' "$$p" $(TESTFLAGS) || exit 1; \
 	done
 
+# Build all tests into DIR and strips each.
+# DIR is required.
+.PHONY: testbuildall
+testbuildall: TESTS := $(shell $(GO) list $(PKG))
+testbuildall: GOFLAGS += -c
+testbuildall:
+ifndef DIR
```

Comment:
I also don't want to rely on test binaries being the only ones named `*.test`. The second someone adds one of those, we'll have a non-binary in the tarball and cause problems.